game/views.py

```
@@ -2272,58 +2272,188 @@ def confirm_delete_account(request, uidb64, token):
+                        score -= val
+                        
+        return score
+
+    played_eval = _simulate_and_evaluate(played_mv)
+    best_eval = _simulate_and_evaluate(best_mv)
+    
+    diff = best_eval - played_eval
+    
+    if diff >= 3:
+        return 'Blunder'
+    elif diff >= 1:
```

Ruff (0.15.20)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)